packages/server/src/ClientConnection.ts

```diff
@@ -301,6 +307,11 @@ export class ClientConnection {
 
       const documentName = decoding.readVarString(tmpMsg.decoder)
 
+      const instance = this.connections[documentName]
+      if (instance) {
+        instance.handleMessage(data)
```

Comment:
and then move this to line 316 :)

Reply:
I think I've made the change you requested here, but I'm unable to see what line 316 is now that I pushed! lol

Please confirm I did what you asked before merging :)

Reply:
Thanks! 🙏